Bychynski_Pavel_HW_2.py

```diff
@@ -0,0 +1,254 @@
+# # 3.1 goal is to find the largest of four numbers
+# # control value (cv)
+# largest_numb = -999999999999999.9
+# # comparing first number with cv
+# numb_1 = float(input("enter your first number: "))
+# if numb_1 > largest_numb:
+# # possible cv change
+#     largest_numb = numb_1
+# # comparing second number with cv
+# numb_2 = float(input("enter your second number: "))
+# if numb_2 > largest_numb:
+# # possible cv change
+#     largest_numb = numb_2
+# # comparing third number with cv
+# numb_3 = float(input("enter your third  number: "))
+# if numb_3 > largest_numb:
+# # possible cv change
+#     largest_numb = numb_3
+# # comparing fourth number with cv
+# numb_4 = float(input("enter your fourth number: "))
+# if numb_4 > largest_numb:
+# # possible cv change
+#     largest_numb = numb_4
+# # result
+# print("you entered: ", numb_1, numb_2, numb_3, numb_4)
+# print("largest number is: ", largest_numb)
```

Comment:
It can be simplified by using: 
a = int(input())
b =..
c = ..
d = ..

and you can use this construction:
if elif elif else